drivers/afe/ad4110/ad4110.c

```diff
@@ -469,6 +542,9 @@ int32_t ad4110_spi_int_reg_read(struct ad4110_dev *dev,
 		buf_size = data_size;
 
 	ret = spi_write_and_read(dev->spi_dev, buf, buf_size);
+	if (ret)
```

Comment:
From here to the bottom, changes seems not to be related with the commit description.